src/__swaps__/screens/Swap/hooks/useSearchCurrencyLists.ts

```diff
@@ -417,7 +417,7 @@ export function useSearchCurrencyLists() {
     {
       enabled: memoizedData.enableUnverifiedSearch,
       select: (data: TokenSearchResult) => {
-        return getExactMatches(data, query).slice(0, MAX_UNVERIFIED_RESULTS);
+        return isAddress(query) ? getExactMatches(data, query).slice(0, MAX_UNVERIFIED_RESULTS) : data.slice(0, MAX_UNVERIFIED_RESULTS);
```

Comment:
without this change, the v3 upgrade was not showing the unverified tokens that "contained" the query (this now matches what we have for BX)